addons/mail_plugin/models/ir_http.py

```diff
@@ -24,3 +24,10 @@ def _auth_method_outlook(cls):
 
         # take the identity of the API key user
         request.uid = user_id
+
+        # set the language of the request to the connected user's language
+        lang = request.env['res.users'].browse(user_id).lang
```

Comment:
Unsure for this one ... @mart-e  maybe ?

Reply:
We have `get_lang` in tools that makes sure the language is installed and fallbacks on the company's language (but not the one of the user) if it helps

Reply:
Thanks for the hint! the current desired behavior is to have the plugin in the language of the User, and not the company's one, so I think that we can keep it like this for now.